src/borg/fuse.py

```diff
@@ -430,6 +438,7 @@ def __init__(self, key, repository, manifest, args, decrypted_repository):
         data_cache_capacity = int(os.environ.get('BORG_MOUNT_DATA_CACHE_ENTRIES', os.cpu_count() or 1))
         logger.debug('mount data cache capacity: %d chunks', data_cache_capacity)
         self.data_cache = LRUCache(capacity=data_cache_capacity, dispose=lambda _: None)
+        self._last_pos = LRUCache(capacity=1, dispose=lambda _: None)
```

Comment:
guess we'll need capacity > 1 if parallel reading of more than a file should perform better?

Reply:
Yes, equal to _read_cache capacity